app/views/sephora_style_guide/home/_icons.slim

```diff
@@ -960,6 +960,26 @@ ruby:
       "size": "icon_90x40"
     }
   ]
+  booking_services_icon = [
+    {
```

Comment:
Could you group together existing booking service icon here e.g `store-white/store-black`.  There might be some others.

Reply:
Moved store-white, store-black and location into the booking service section

Reply:
Where is location icon is being used in booking services?